fastapitableau/statics/css/styles.css

```diff
@@ -1,76 +1,520 @@
+@charset "UTF-8";
+html {
```

Comment:
![image](https://user-images.githubusercontent.com/3117884/134353518-245af51a-494d-48c3-a5d5-2dd4618933a0.png)

Some of our code styling has been reverted or is missing. I'd made some changes in the last PR — copying font strings and padding from GitHub. I'm not sure if that was an intentional change, or something that got accidentally pulled out in the accidental CSS stripping.

Either way, happy to work on it some more. It'll be much easier with this more streamlined setup. 👍🏻